test/integration/smoke-testing/test_signing_in_and_out_discuss.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Tests from:
+ *
+ * https://github.com/LLK/scratchr2/wiki/Smoke-Testing-Test-Cases
+ *
+ */
+
+const {
+    clickText,
+    findByXpath,
+    findText,
+    clickXpath,
+    clickButton,
+    driver
+} = require('../../helpers/selenium-helpers.js');
+
+var username = process.env.SMOKE_USERNAME;
+var password = process.env.SMOKE_PASSWORD;
```

Comment:
Are these environment variables already documented somewhere? If not we should add them.